conduct.rst

```diff
@@ -0,0 +1,252 @@
+
+The first rule means
+you shouldn't act surprised when people say
+they don't know something.
+This applies to both technical things
+("What?! I can't believe you don't know what the stack is!")
+and non-technical things ("You don't know who RMS is?!").
+Feigning surprise has absolutely
+no social or educational benefit:
+When people feign surprise,
+it's usually to make them feel better about themselves
+and others feel worse.
+And even when that's not the intention,
+it's almost always the effect.
+This rule is tightly coupled
+to our belief in the importance of people
+feeling comfortable saying "I don't know" and "I don't understand."
+
+No condescending well-actually's
+--------------------------------
+
+A well-actually happens
+when someone says something that's almost—but not entirely—correct,
+and you say, "well, actually..." and then give a minor correction.
+Well-actually gives the impression
+that a mostly correct description is fully wrong,
+which is rarely the case.
+Think about whether the correction
+is critical to to the conversation.
+If it is, phrasing can be the difference
+between a valuable clarification and condescension;
+instead of "well, actually..." try
+"that's right, but don't forget..."
+or "in addition to that description, it's also true that..."
+
+No backseat driving
+-------------------
+
+If a thread is working through a problem,
+you shouldn't drop into the thread,
+drop an advice bomb, then leave.
+This can lead to the "too many cooks" problem,
+but more importantly,
+it can be rude and disruptive
+to half-participate in a conversation.
+This isn't to say you shouldn't help,
+offer advice, or join conversation.
+On the contrary, we encourage all those things.
+Rather, it means that when you want to
+help out or work with others,
+you should fully engage
+and not just interject sporadically.
+If you want to show support
+for an idea presented in a thread,
+consider using a simple thumbs up emoji reaction
+rather than stating the same point a different way.
+If you feel that there is a different approach
+that could be taken,
+but you do not have the time or motivation
+to implement that approach yourself,
+make a note to yourself to explore that approach
+when you have time
+instead of putting that burden on someone else.
+
+No subtle -isms
+---------------
+
+Subtle -isms, also called
+`microaggressions <http://geekfeminism.wikia.com/wiki/Microaggressions>`_,
+are small things that make others feel uncomfortable.
+For example, saying "it's so easy my grandmother could do it" is a subtle -ism,
+as it is both subtly sexist and ageist.
+The "subtle" in "subtle -isms" means that it's probably
+not obvious to everyone right away what was wrong with the comment,
+even to people in the group otherwise affected by the comment.
+And, even though they are subtle,
+might seem insignificant,
+and are often unintentional,
+a steady stream of them compounds
+to make people in under-represented groups feel less welcome.
+
```

Comment:
again, it could just be because the example here is a little over-the-top, but i don't think this kind of thing is happening or likely to happen; strikes me similarly to above on deleting this if we want to shorten.